db/migrate/20190102190552_create_leads.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLeads < ActiveRecord::Migration[5.2]
+  def change
+    create_table :leads do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :phone_number
```

Comment:
Might also want to have some sort of `status` flag on this so that (eventually) the admin can mark as resolved. Could also wait until you get to that part in ActiveAdmin.

Reply:
will wait for active admin and to find out how client wants to handle it